toolium/utils/dataset.py

```diff
@@ -813,3 +818,33 @@ def convert_file_to_base64(file_path):
     except Exception as e:
         raise Exception(f' ERROR - converting the "{file_path}" file to Base64...: {e}')
     return file_content
+
+
+def store_key_in_storage(context, key, value):
+    """
+    Store values in context.storage, context.feature_storage or context.run_storage,
+    using [key], [FEATURE:key] OR [RUN:key] from steps.
+    context.storage is also updated with given key,value
+    By default, values are stored in context.storage.
+
+    :param key: key to store the value in proper storage
+    :param value: value to store in key
+    :param context: behave context
+    :return:
+    """
+    clean_key = re.sub(r'[\[\]]', '', key)
+    if ":" in clean_key:
+        context_type = clean_key.split(":")[0]
+        context_key = clean_key.split(":")[1]
+        acccepted_context_types = ["FEATURE", "RUN"]
+        assert context_type in acccepted_context_types, (f"Invalid key: {context_key}. "
+                                                         f"Accepted keys: {acccepted_context_types}")
+        if context_type == "RUN":
+            context.run_storage[context_key] = value
+        elif context_type == "FEATURE":
+            context.feature_storage[context_key] = value
+        # If dynamic env is not initialized linked or key exists in context.storage, the value is updated in it
+        if hasattr(context.storage, context_key) or not isinstance(context.storage, collections.ChainMap):
+            context.storage[context_key] = value
+    else:
+        context.storage[clean_key] = value
```

Comment:
Info: This function should always be used by the programmer if the expected behaviour is to use feature and run storage. Otherwise it's the programmer responsability to handle and avoid duplicated keys, etc if they use directly the context.storage[key]=x and context.feature_storage[key]=x

We also kept the FEATURE: and RUN: tags as it gives more flexibility.

We will also add in toolium_telefonica the steps to replicate any value to desired storage:
I store the value "" in feature storage with key ""
-> Erased from storage,run_storage  and moved to feature_storage

I store the value "" in run storage with key ""
-> Erased from storage, feature_storage and moved to run_storage


This allowas us with a simple step to "replicate" or "move" values from storages